bilby/gw/detector/calibration.py

```diff
@@ -37,6 +46,21 @@ def read_calibration_file(filename, frequency_array, number_of_response_curves,
     """
     import tables
 
+    if correction is None:
+        logger.warning(
+            "Calibration envelope correction type is not specified. "
+            "Assuming this correction maps calibrated data to theoretical "
+            "strain. If this is correct, this should be explicitly "
+            "specified via CalibrationPriorDict.from_envelope_file(..., "
+            "correction='data')."
```

Comment:
Perhaps we should also discuss what to do when `correction='data'` is not the intended operation?